Binary files /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/index and /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/index differ
diff -ru /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/logs/HEAD /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/logs/HEAD
--- /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/logs/HEAD	2020-11-27 14:46:12.101878838 +0100
+++ /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/logs/HEAD	2020-11-27 14:46:33.493860260 +0100
@@ -1 +1 @@
-0000000000000000000000000000000000000000 3343810d7eb396b48a00d22a6e9964adbde2e6b1 Benjamin DANGLOT <benjamin.danglot@davidson.fr> 1606484772 +0100	commit (initial): Initial commit
+0000000000000000000000000000000000000000 605af7eabfa19ff25f76a3ade3a9e027762a9b71 Benjamin DANGLOT <benjamin.danglot@davidson.fr> 1606484793 +0100	commit (initial): Initial commit
diff -ru /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/logs/refs/heads/master /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/logs/refs/heads/master
--- /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/logs/refs/heads/master	2020-11-27 14:46:12.101878838 +0100
+++ /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/logs/refs/heads/master	2020-11-27 14:46:33.493860260 +0100
@@ -1 +1 @@
-0000000000000000000000000000000000000000 3343810d7eb396b48a00d22a6e9964adbde2e6b1 Benjamin DANGLOT <benjamin.danglot@davidson.fr> 1606484772 +0100	commit (initial): Initial commit
+0000000000000000000000000000000000000000 605af7eabfa19ff25f76a3ade3a9e027762a9b71 Benjamin DANGLOT <benjamin.danglot@davidson.fr> 1606484793 +0100	commit (initial): Initial commit
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/objects: 0a
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/objects: 21
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/objects: 29
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/objects: 2c
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/objects: 33
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/objects: 3a
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/objects: 44
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/objects: 4b
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/objects: 55
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/objects: 56
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/objects: 60
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/objects: 64
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/objects: 6c
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/objects/71: 3d433f832e955ec3a3438dacc027a643f51f39
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/objects/71: fb28a3dd33fdc4c9238526997defa45ca30635
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/objects: 7b
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/objects: 7e
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/objects: 83
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/objects: 87
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/objects: 89
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/objects: 91
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/objects: a4
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/objects: a7
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/objects: ba
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/objects: c1
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/objects/cc: 17f7ad9cb1661777c75e4b771a889a2f97c4ec
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/objects/cc: 6d39b7859ea9574737095bce4c36b285af7745
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/objects: d6
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/objects: da
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/objects: e1
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/objects: f6
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/objects: fb
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/objects: fe
diff -ru /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/refs/heads/master /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/refs/heads/master
--- /home/benjamin/workspace/diff-jjoules/src/test/resources/example/.git/refs/heads/master	2020-11-27 14:46:12.101878838 +0100
+++ /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/.git/refs/heads/master	2020-11-27 14:46:33.493860260 +0100
@@ -1 +1 @@
-3343810d7eb396b48a00d22a6e9964adbde2e6b1
+605af7eabfa19ff25f76a3ade3a9e027762a9b71
diff -ru /home/benjamin/workspace/diff-jjoules/src/test/resources/example/pom.xml /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/pom.xml
--- /home/benjamin/workspace/diff-jjoules/src/test/resources/example/pom.xml	2020-11-27 13:39:37.075335011 +0100
+++ /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/pom.xml	2020-11-26 15:50:51.719000000 +0100
@@ -50,70 +50,70 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.22.2</version>
             </plugin>
-<!--            <plugin>-->
-<!--                <artifactId>maven-failsafe-plugin</artifactId>-->
-<!--                <version>2.22.2</version>-->
-<!--            </plugin>-->
+            <plugin>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <version>2.22.2</version>
+            </plugin>
 
-<!--            &lt;!&ndash; Configuration for call graph &ndash;&gt;-->
-<!--            <plugin>-->
-<!--                <groupId>org.apache.maven.plugins</groupId>-->
-<!--                <artifactId>maven-jar-plugin</artifactId>-->
-<!--                <version>3.2.0</version>-->
-<!--                <executions>-->
-<!--                    <execution>-->
-<!--                        <phase>test</phase>-->
-<!--                        <goals>-->
-<!--                            <goal>test-jar</goal>-->
-<!--                        </goals>-->
-<!--                        &lt;!&ndash; <configuration> <excludes> <exclude>org/powerapi/jjoules/Fibonacci.class</exclude>-->
-<!--                            </excludes> </configuration> &ndash;&gt;-->
-<!--                    </execution>-->
-<!--                </executions>-->
-<!--            </plugin>-->
+            <!-- Configuration for call graph -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>3.2.0</version>
+                <executions>
+                    <execution>
+                        <phase>test</phase>
+                        <goals>
+                            <goal>test-jar</goal>
+                        </goals>
+                        <!-- <configuration> <excludes> <exclude>org/powerapi/jjoules/Fibonacci.class</exclude>
+                            </excludes> </configuration> -->
+                    </execution>
+                </executions>
+            </plugin>
 
-<!--            <plugin>-->
-<!--                <groupId>org.codehaus.mojo</groupId>-->
-<!--                <artifactId>exec-maven-plugin</artifactId>-->
-<!--                <version>3.0.0</version>-->
-<!--                <executions>-->
-<!--                    <execution>-->
-<!--                        <id>call-graph</id>-->
-<!--                        <phase>test</phase>-->
-<!--                        <goals>-->
-<!--                            <goal>exec</goal>-->
-<!--                        </goals>-->
-<!--                        <configuration>-->
-<!--                            <executable>java</executable>-->
-<!--                            <arguments>-->
-<!--                                <argument>-jar</argument>-->
-<!--                                <argument>javacg-0.1-SNAPSHOT-static.jar</argument>-->
-<!--                                <argument>${project.build.directory}/${project.build.finalName}-tests.jar</argument>-->
-<!--                            </arguments>-->
-<!--                            <outputFile>${project.build.directory}/call-graph/tests-callgraph.txt</outputFile>-->
-<!--                        </configuration>-->
-<!--                    </execution>-->
-<!--                </executions>-->
-<!--            </plugin>-->
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>3.0.0</version>
+                <executions>
+                    <execution>
+                        <id>call-graph</id>
+                        <phase>test</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>java</executable>
+                            <arguments>
+                                <argument>-jar</argument>
+                                <argument>javacg-0.1-SNAPSHOT-static.jar</argument>
+                                <argument>${project.build.directory}/${project.build.finalName}-tests.jar</argument>
+                            </arguments>
+                            <outputFile>${project.build.directory}/call-graph/tests-callgraph.txt</outputFile>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
 
-<!--            <plugin>-->
-<!--                <groupId>org.powerapi.jjoules</groupId>-->
-<!--                <artifactId>junit-jjoules</artifactId>-->
-<!--                <version>1.0-SNAPSHOT</version>-->
-<!--                <executions>-->
-<!--                    <execution>-->
-<!--                        <id>callgraph-matrix</id>-->
-<!--                        <phase>test</phase>-->
-<!--                        <goals>-->
-<!--                            <goal>generate-matrix</goal>-->
-<!--                        </goals>-->
-<!--                        &lt;!&ndash; example configuration &ndash;&gt;-->
-<!--                        &lt;!&ndash; <configuration> <outputDir>${project.build.directory}/tests-call-graphs</outputDir>-->
-<!--                            <outputFilename>matrix.txt</outputFilename> <callgraphFilename>callgraph.txt</callgraphFilename>-->
-<!--                            </configuration> &ndash;&gt;-->
-<!--                    </execution>-->
-<!--                </executions>-->
-<!--            </plugin>-->
+            <plugin>
+                <groupId>org.powerapi.jjoules</groupId>
+                <artifactId>junit-jjoules</artifactId>
+                <version>1.0-SNAPSHOT</version>
+                <executions>
+                    <execution>
+                        <id>callgraph-matrix</id>
+                        <phase>test</phase>
+                        <goals>
+                            <goal>generate-matrix</goal>
+                        </goals>
+                        <!-- example configuration -->
+                        <!-- <configuration> <outputDir>${project.build.directory}/tests-call-graphs</outputDir>
+                            <outputFilename>matrix.txt</outputFilename> <callgraphFilename>callgraph.txt</callgraphFilename>
+                            </configuration> -->
+                    </execution>
+                </executions>
+            </plugin>
         </plugins>
     </build>
 </project>
diff -ru /home/benjamin/workspace/diff-jjoules/src/test/resources/example/src/main/java/fr/davidson/example/Fibonacci.java /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/src/main/java/fr/davidson/example/Fibonacci.java
--- /home/benjamin/workspace/diff-jjoules/src/test/resources/example/src/main/java/fr/davidson/example/Fibonacci.java	2020-11-27 14:28:12.610538089 +0100
+++ /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/src/main/java/fr/davidson/example/Fibonacci.java	2020-11-27 14:28:28.602536192 +0100
@@ -17,7 +17,7 @@
         } else if (n == 1) {
             return b;
         } else {
-            Thread.sleep(100);
+            Thread.sleep(1);
             return fibInner(b, a + b, n - 1);
         }
     }
diff -ru /home/benjamin/workspace/diff-jjoules/src/test/resources/example/src/test/java/fr/davidson/example/FibonacciTest.java /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/src/test/java/fr/davidson/example/FibonacciTest.java
--- /home/benjamin/workspace/diff-jjoules/src/test/resources/example/src/test/java/fr/davidson/example/FibonacciTest.java	2020-11-27 14:58:34.621562984 +0100
+++ /home/benjamin/workspace/diff-jjoules/src/test/resources/example/../example-refactor/src/test/java/fr/davidson/example/FibonacciTest.java	2020-11-27 15:43:03.987194479 +0100
@@ -1,16 +1,16 @@
 package fr.davidson.example;
-
 import org.junit.jupiter.api.Test;
-
+import org.powerapi.jjoules.junit.EnergyTest;
+import org.junit.jupiter.api.Test;
 import static org.junit.jupiter.api.Assertions.assertEquals;
-
 /**
+ *
+ *
  * @author Benjamin DANGLOT
- * benjamin.danglot@davidson.fr
- * 26/11/2020
+benjamin.danglot@davidson.fr
+26/11/2020
  */
 public class FibonacciTest {
-
     @Test
     void testZero() throws InterruptedException {
         assertEquals(0, Fibonacci.fibTailRec(0));
@@ -21,14 +21,14 @@
         assertEquals(1, Fibonacci.fibTailRec(1));
     }
 
-    @Test
+    @EnergyTest
     void test() throws InterruptedException {
         assertEquals(28657, Fibonacci.fibTailRec(23));
     }
 
-    @Test
+    @EnergyTest
     void testLarge() throws InterruptedException {
         final long oracle = 8394940206042357789L;
         assertEquals(oracle, Fibonacci.fibTailRec(Byte.MAX_VALUE));
     }
-}
+}
\ No newline at end of file
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example: testsThatExecuteTheChange_coverage.csv
Only in /home/benjamin/workspace/diff-jjoules/src/test/resources/example: testsThatExecuteTheChange.csv
